components/MapContainer/MapContainer.tsx

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import { Select } from '../Select';
+import { Map } from '../Map';
+import ugandaDistricts from '../../geoJSON/district.json';
+import ugandasubcounties from '../../geoJSON/subcounty.json';
```

Comment:
Again, unless this component is going to be used by only Uganda code, this should be here. Either make a prop, or rename this component to be UgandaMapContainer or something. I also don't think it's a good idea to mix Uganda & Kenya code ... I suspect their GeoJSON is quite different, but that aside, it's going to needlessly crowd things.

Reply:
Renamed to `UgandaContainer`